docs/core/extensions/service-discovery.md

```diff
@@ -135,12 +135,12 @@ builder.Services.AddHttpClient<CatalogServiceClient>(
 
 The `Microsoft.Extensions.ServiceDiscovery` package includes the following endpoint selector providers:
 
-- Pick-first, which always selects the first endpoint: `PickFirstServiceEndPointSelectorProvider.Instance`
-- Round-robin, which cycles through endpoints: `RoundRobinServiceEndPointSelectorProvider.Instance`
-- Random, which selects endpoints randomly: `RandomServiceEndPointSelectorProvider.Instance`
-- Power-of-two-choices, which attempt to pick the least heavily loaded endpoint based on the _Power of Two Choices_ algorithm for distributed load balancing, degrading to randomly selecting an endpoint when either of the provided endpoints don't have the `IEndPointLoadFeature` feature: `PowerOfTwoChoicesServiceEndPointSelectorProvider.Instance`
+- <xref:Microsoft.Extensions.ServiceDiscovery.Abstractions.PickFirstServiceEndPointSelectorProvider.Instance?displayProperty=nameWithType>: Pick-first, which always selects the first endpoint.
+- <xref:Microsoft.Extensions.ServiceDiscovery.Abstractions.RoundRobinServiceEndPointSelectorProvider.Instance?displayProperty=nameWithType>: Round-robin, which cycles through endpoints.
+- <xref:Microsoft.Extensions.ServiceDiscovery.Abstractions.RandomServiceEndPointSelectorProvider.Instance?displayProperty=nameWithType>: Random, which selects endpoints randomly.
+- <xref:Microsoft.Extensions.ServiceDiscovery.Abstractions.PowerOfTwoChoicesServiceEndPointSelectorProvider.Instance?displayProperty=nameWithType>: Power-of-two-choices, which attempt to pick the least heavily loaded endpoint based on the _Power of Two Choices_ algorithm for distributed load balancing, degrading to randomly selecting an endpoint when either of the provided endpoints don't have the <xref:Microsoft.Extensions.ServiceDiscovery.Abstractions.IEndPointLoadFeature> feature.
```

Comment:
Is there a better phrase than "least heavily loaded?"
- least used
- underloaded
- lightest load
- 
?